webpack.config.js

```diff
@@ -130,6 +149,13 @@ const classicEditor = {
 	entry: {
 		'amp-post-meta-box': './assets/src/classic-editor/amp-post-meta-box.js',
 	},
+	plugins: [
+		...defaultConfig.plugins,
+		new WebpackBar( {
+			name: 'Block Editor',
```

Comment:
@swissspidy  Just in case -- is it intended for this to be `Block Editor` and not `Classic Editor`?

Reply:
D‘oh, yes. Copy/Paste mistake.